### PR TITLE
feat: tup-463 migrate css to core

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Migrate CSS from here to Core-CMS and Core-Styles.

## Status

Tested many times. Fixed many bugs. Ready for review.

<details>

- [x] deploy 0e84c10 — Build 134 (Nov 13, 2023 11:13:35 AM)
- [x] deploy 3167014 — Build 135 (Nov 13, 2023 11:47:59 AM)
- [x] bugs [TUP-646](https://jira.tacc.utexas.edu/browse/TUP-646), [TUP-649](https://jira.tacc.utexas.edu/browse/TUP-649), [TUP-650](https://jira.tacc.utexas.edu/browse/TUP-650), [TUP-654](https://jira.tacc.utexas.edu/browse/TUP-654)
- [x] deploy bee6a46 — Build 136 (Nov 20, 2023 7:09:34 PM)
- [x] test previous bug fixes
- [x] bugs [TUP-657](https://jira.tacc.utexas.edu/browse/TUP-657), [TUP-658](https://jira.tacc.utexas.edu/browse/TUP-658), [TUP-659](https://jira.tacc.utexas.edu/browse/TUP-659), [TUP-660](https://jira.tacc.utexas.edu/browse/TUP-660)
- [x] install new core-styles (and core-cms with new core-styles)
- [x] deploy — Build 137 (Nov 21, 2023 6:03:53 PM)
- [x] test
- [x] deploy — Build 138 (Nov 22, 2023 1:12:14 PM)
- [x] test (after turkey time)
- [x] deploy — Nov 27
- [x] test
- [x] deploy Core-CMS at a release (v4.2.0) — Build 142 (Nov 28, 2023 2:47:44 PM)

</details>

## Related

- [TUP-463](https://jira.tacc.utexas.edu/browse/TUP-463)
- requires https://github.com/TACC/Core-CMS/pull/742

## Changes

- **removed** styles migrated to Core
- **changed** Core-CMS image tag
- **changed** Core-Styles version

## Testing & UI

This is mostly moving styles and fixing bugs, so UI looks the same. But there are some differences.

See [merged PRs to `feat/tup-463` branch before 2023-12-01](https://github.com/TACC/tup-ui/pulls?q=is%3Apr+is%3Amerged+label%3A%22migrate+css+to+core%22+base%3Afeat%2Ftup-463+closed%3A%3C2023-12-01).

| change | screenshot |
| - | - |
| search is styled | search
| footer buttons less negative space | footer buttons
| form button is uppercase | form button uppercase
| "Learn more" links wrap | learn more links wrap

### CODE REVIEW

File: apps/tup-cms/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# TACC/Core-CMS#735 (cms-forms export hotfix)
-FROM taccwma/core-cms:5ceecdd
+# TACC/Core-CMS#742 (Core-Styles v2.11 to v2.19)
```
Comment (wesleyboar) [inline apps/tup-cms/Dockerfile:1]:
```suggestion
# TACC/Core-CMS#742 (core-styles v2.11 to v2.19)
```

**Comment by wesleyboar** [comment]
> [!WARNING]
> MFA `c-nav` is broken. #375 is suspect (CSS module was removed in favor of Core-Styles CSS).
> mfa c-nav broke

**Review by sophia-massie — APPROVED** [review]
Looks great! Thank you

**Comment by wesleyboar** [comment]
Merging with one approval, because:

- @R-Tomas-Gonzalez coded most of this, and I approved his work, and he helped me re-test afterwards.
- @jarosenb is pleased with mass removal of code, and generally trusts me for UI.
- I take responsibility for any UI breaking.
- I tested well, and fixed things.

**Pull request merged**